### PR TITLE
table centered

### PR DESCRIPTION
**Pull request by himanshu007-creator**
i converted markdown table to html one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,37 @@
...
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/Manual%20workflow/badge.svg"></td>
+  </tr>
+  <tr>
+    <td>New Contributors</td>
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/Greetings/badge.svg"></td>
+  </tr>
+</tbody>
+</table>
+</div>
+
```
Comment (IIITM-Jay) [inline README.md:33]: @himanshu007-creator Himanshu, this is simply an HTML code, can you try using the markdown format as it is the standard format for writing the readme files.

**Review by IIITM-Jay — CHANGES_REQUESTED** [review]
Himanshu @himanshu007-creator try  using markdown as well as flavored markdown

**Comment by himanshu007-creator** [comment]
> Himanshu @himanshu007-creator try using markdown as well as flavored markdown

Okay sir, will try it today

**Comment by himanshu007-creator** [comment]
i am working on it sir, but i don't think there is a way to align a table created in markdown in center.

**Comment by himanshu007-creator** [comment]
sir, i dont think there is a way to center align a table created in markdown. I have checked many discussions on many platforms, but all proved useless. So HTML table is the only solution.

**Comment by IIITM-Jay** [comment]
@himanshu007-creator Himanshu okay , so go ahead with updating the PR with a new file as the last page contains some errors...

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Himanshu okay , so go ahead with updating the PR with a new file as the last page contains some errors...

I have corrected them. Please check

**Comment by IIITM-Jay** [comment]
@himanshu007-creator Himanshu once again see by adding a new file. Do one thing do not make any changes, upload the unchanged file (with the same file that is already present in this repo without any changes done by you) from your side again and will get merged soon.

**Comment by IIITM-Jay** [comment]
Soory himanshu @himanshu007-creator but I have been asked to label it as level 1 that's why I changed it to that..

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Himanshu once again see by adding a new file. Do one thing do not make any changes, upload the unchanged file (with the same file that is already present in this repo without any changes done by you) from your side again and will get merged soon.

done sir

**Comment by IIITM-Jay** [comment]
Himanshu @himanshu007-creator If you want this to be labeled as level 3 or 4, can you create two bages namely frontend view and backend view that will link to the same. The link I will provide you late but first can you try building up badges as the one you are seeing at the top of the **Readme.md** file.

**Comment by himanshu007-creator** [comment]
> Himanshu @himanshu007-creator If you want this to be labeled as level 3 or 4, can you create two badges namely frontend view and backend view that will link to the same. The link I will provide you later , so don't worry about the links but first can you try building up badges as the one you are seeing at the top of the **Readme.md** file and show me the designed badges.

Yes, I can make them sir

**Comment by himanshu007-creator** [comment]
> Himanshu @himanshu007-creator If you want this to be labeled as level 3 or 4, can you create two badges namely frontend view and backend view that will link to the same. The link I will provide you later , so don't worry about the links but first can you try building up badges as the one you are seeing at the top of the **Readme.md** file and show me the designed badges.

please have a look now 😃

**Comment by IIITM-Jay** [comment]
@himanshu007-creator Himanshu look it once again the file you have changed ... you have not yet cleared unwanted...

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Himanshu look it once again the file you have changed ... you have not yet cleared unwanted...

sir, can you tell which lines exactly?

**Comment by himanshu007-creator** [comment]
> Himanshu @himanshu007-creator If you want this to be labeled as level 3 or 4, can you create two badges namely frontend view and backend view that will link to the same. The link I will provide you later , so don't worry about the links but first can you try building up badges as the one you are seeing at the top of the **Readme.md** file and show me the designed badges.

did this too, so i think now labels should be made <code> level-3</code> or <code> level-4</code>

**Comment by IIITM-Jay** [comment]
@himanshu007-creator sorry to say Himanshu but this can go as level 2 when I asked and cleared the same in detail... I found so

**Comment by IIITM-Jay** [comment]
@himanshu007-creator Himanshu is it okay with you...

**Comment by IIITM-Jay** [comment]
@Nandini-Jha I think this can be merged now :hugs:

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Himanshu is it okay with you...

totally fine!

**Comment by IIITM-Jay** [comment]
@himanshu007-creator Great work Himanshu... :tada: :sparkles: 
Thank you so much for your contributions.
Once gain welcoming you and cogratulating to be a part of the Property Management System :house_with_garden:

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Great work Himanshu... 🎉 ✨
> Thank you so much for your contributions.
> Once gain welcoming you and cogratulating to be a part of the Property Management System 🏡

welcome sir!

**Comment by IIITM-Jay** [comment]
@all-contributors please add @himanshu007-creator  for content

**Pull request merged**